transports/noise/src/io/handshake.rs

```diff
@@ -31,8 +32,10 @@ use crate::protocol::{KeypairIdentity, STATIC_KEY_DOMAIN};
 use crate::{DecodeError, Error};
 use bytes::Bytes;
 use futures::prelude::*;
+use libp2p_core::multihash::Multihash;
```

Comment:
@thomaseizinger long term, should crates depend on `multihash` through `libp2p-core` or define it as an explicit dependency in their `Cargo.toml`. I am fine either way for this pull request. Not blocking.

Reply:
> @thomaseizinger long term, should crates depend on `multihash` through `libp2p-core` or define it as an explicit dependency in their `Cargo.toml`. I am fine either way for this pull request. Not blocking.

Long-term, they should depend on it themselves. I want to eventually remove this re-export and/or dissolve `libp2p-core` completely.

Reply:
I now use `multiaddr` and `multihash` directly